### PR TITLE
Manager large backup scenario

### PR DESCRIPTION
**Pull request by ShlomiBalalis**
## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Comment by soyacz** [comment]
@ShlomiBalalis make sure you provide descriptive commit messages (all commits) and some description for the PR itself.

**Comment by ShlomiBalalis** [comment]
> @ShlomiBalalis make sure you provide descriptive commit messages (all commits) and some description for the PR itself.

Of course, it's just in draft for now

File: test-cases/manager/manager-regression-multiDC-large-set-distro.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+test_duration: 1440
+
+prepare_write_cmd: ["cassandra-stress write cl=QUORUM n=550100150 -schema 'replication(factor=3)' -mode cql3 native connectionsPerHost=64 -rate threads=500 throttle=45000/s -col 'size=FIXED(200) n=FIXED(5)' -pop seq=1..550100150",
```
Comment (roydahan) [inline test-cases/manager/manager-regression-multiDC-large-set-distro.yaml:3]: @ShlomiBalalis why are you using throttle here?
It's slowing down the prepare.

Also, Are you trying to write 1TB? why not splitting to 4 loaders and writing faster.

File: test-cases/manager/manager-regression-multiDC-large-set-distro.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+stress_read_cmd: ["cassandra-stress read cl=QUORUM n=550100150 -schema 'replication(factor=3)' -mode cql3 native connectionsPerHost=64 -rate threads=500 throttle=45000/s -col 'size=FIXED(200) n=FIXED(5)' -pop seq=1..550100150",
+                  "cassandra-stress read cl=QUORUM n=550100150 -schema 'replication(factor=3)' -mode cql3 native connectionsPerHost=64 -rate threads=500 throttle=45000/s -col 'size=FIXED(200) n=FIXED(5)' -pop seq=550100151..1100200300",
+                  ]
+round_robin: true
+data_volume_disk_size: 1500
+
+instance_type_db: 'i3.2xlarge'
+instance_type_loader: 'c5.large'
+
+region_name: 'us-east-1 us-west-2'
```
Comment (roydahan) [inline test-cases/manager/manager-regression-multiDC-large-set-distro.yaml:15]: Is there a value to test it with multi-DC?

File: test-cases/manager/manager-regression-multiDC-large-set-distro.yaml (hunk 1 above)
Comment (ShlomiBalalis) [reply test-cases/manager/manager-regression-multiDC-large-set-distro.yaml:3]: This PR is outdated, I'll update it shortly

File: test-cases/manager/manager-regression-multiDC-large-set-distro.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
+test_duration: 1440
+
+prepare_write_cmd: ["cassandra-stress write cl=QUORUM n=550100150 -schema 'replication(factor=3)' -mode cql3 native connectionsPerHost=64 -rate threads=500 throttle=45000/s -col 'size=FIXED(200) n=FIXED(5)' -pop seq=1..550100150",
+                    "cassandra-stress write cl=QUORUM n=550100150 -schema 'replication(factor=3)' -mode cql3 native connectionsPerHost=64 -rate threads=500 throttle=45000/s -col 'size=FIXED(200) n=FIXED(5)' -pop seq=550100151..1100200300",
+                    ]
+stress_read_cmd: ["cassandra-stress read cl=QUORUM n=550100150 -schema 'replication(factor=3)' -mode cql3 native connectionsPerHost=64 -rate threads=500 throttle=45000/s -col 'size=FIXED(200) n=FIXED(5)' -pop seq=1..550100150",
+                  "cassandra-stress read cl=QUORUM n=550100150 -schema 'replication(factor=3)' -mode cql3 native connectionsPerHost=64 -rate threads=500 throttle=45000/s -col 'size=FIXED(200) n=FIXED(5)' -pop seq=550100151..1100200300",
+                  ]
+round_robin: true
+data_volume_disk_size: 1500
```
Comment (roydahan) [inline test-cases/manager/manager-regression-multiDC-large-set-distro.yaml:10]: you don't need this!

**Pull request closed without merging**